irohad/torii/processor/transaction_processor.hpp

```diff
@@ -39,12 +39,6 @@ namespace iroha {
       virtual void transaction_handle(model::Client client,
                                       model::Transaction &transaction) = 0;
 
-      /**
-       * Subscribers will be notified with transaction status
-       * @return observable for subscribing
-       */
-      virtual rxcpp::observable<model::TransactionResponse>
-      transaction_notifier() = 0;
```

Comment:
sorry, that is not my commit.
I checkouted this branch (https://github.com/hyperledger/iroha/pull/427)